aten/src/ATen/core/ivalue.cpp

```diff
@@ -204,6 +204,9 @@ void ivalue::Object::resizeObject(size_t slot) {
   slots_.resize(type()->numAttributes());
 }
 
+
+ivalue::Future::Future() : type_(c10::NoneType::get()) {}
```

Comment:
Should not be part of this PR.

Reply:
yea, sorry -- I mentioned that in an above comment (I didn't use ghstack for this). will go away